sbin/submitty_daemon_jobs/submitty_jobs/bulk_qr_split.py

```diff
@@ -33,15 +33,15 @@ def main(args):
 
     try:
         os.chdir(split_path)
-        pdfPages = PdfFileReader(filename)
+        pdfPages = PdfFileReader(filename, strict=False)
         pdf_writer = PdfFileWriter()
```

Comment:
We'll need to update the class names while we're at it as the `PdfFile*` version has been deprecated in favor of the shorter names.

```suggestion
        pdfPages = PdfReader(filename, strict=False)
        pdf_writer = PdfWriter()
```